test-avocado/lib/testvm.py

```diff
@@ -0,0 +1 @@
+../../test/testvm.py
```

Comment:
Let's hope that we get rid of the old tests quickly.  Then we can move testlib.py into test-avocado and clean it up.